frontend/src/components/Admin/TeamEvents.tsx

```diff
@@ -15,6 +16,14 @@ export type TeamEvent = {
   uuid: string;
 };
 
+type TeamEventAttendance = {
```

Comment:
I think having another type for team event attendance is definitely good since there can be varying hours attended for some events. We can probably change the `TeamEvent` type to 
```
interface TeamEvent {
    name: string;
    date: string;
    numCredits: string;
    hasHours: boolean;
    *** readonly attendees: TeamEventAttendance[]; ***
    *** readonly requests: TeamEventAttendance[]; ***
    readonly uuid: string;
}
```
And then maybe just have the `TeamEventAttendance` type be 
```
interface TeamEventInterface {
    member: IdolMember;
    hoursAttendend?: number;
    image: string;
}
```
This way we don't need an id or team event name associated with the `TeamEventAttendance` type since they're associated with a `TeamEvent` type.